docs/deploy_streamlit_app.md

```diff
@@ -84,11 +84,26 @@ Streamlit sharing is completely free and is meant to get you started with sharin
 
 ### Access
 
-- Apps are always visible to the entire world.
+- Apps are visible to the entire world.
 - You can only deploy apps that are in a public GitHub repo.
 - Your source code must live in Github. We're looking to expand to other Git hosts soon.
 - Everyone with push access to your repo is automatically a maintainer of the app.
 
+### Long Term
+
+For apps that have been deployed for a long time but are not getting any views or traffic, they will automatically go to sleep after 7 days. This is done to alleviate resources and allow the best communal use of the platform! Here are some need to know's about how this works:
```

Comment:
"For apps that have been deployed a long time" is unnecessary here. The sentence can easily be shortened to "For apps without traffic for 7 consecutive days, they will automatically go to sleep. This is done..." without a loss of meaning

Reply:
I used your shorter sentence here